model.py

```diff
@@ -5785,10 +5794,16 @@ class CachedFeed(Base):
     # The content of the feed.
     content = Column(Unicode, nullable=True)
 
-    # A feed may be associated with a LicensePool.
+    # TODO: After LicensePool-associated feeds have been successfully
+    # migrated to Work-associated feeds, drop this column and remove
+    # its relationship in LicensePool.
```

Comment:
This only impacts Recommendations (which aren't configured in QA or production, I think) and their parent Related Books lanes.